tests/test_discretized_curves.py

```diff
@@ -256,7 +257,9 @@ def test_srv_metric_dist_and_geod(self):
         srv = self.srv_metric_r3.square_root_velocity(geod)
 
         srv_derivative = self.n_discretized_curves * (srv[1:, :] - srv[:-1, :])
-        norms = self.l2_metric_r3.norm(srv_derivative, geod[:-1, :-1, :])
+        l2_metric = self.space_curves_in_euclidean_3d.l2_metric(
+            self.n_sampling_points - 1)
```

Comment:
Why do we need `self.n_sampling_points -1` here, and in setup we have:
```
self.l2_metric_r3 = self.space_curves_in_euclidean_3d.l2_metric(
            self.n_sampling_points)
```

Reply:
in the setup it is used as the L2 metric, here it is used to test the SRV metric, which results in `n_sampling_points - 1` points